stubs/openpyxl/openpyxl/drawing/image.pyi

```diff
@@ -1,9 +1,14 @@
-from _typeshed import Incomplete
+from typing_extensions import Literal
+
+from PIL import Image as ImageModule
+
+# Using "object" because modules can't by used as type
+PILImage: object | Literal[False]
```

Comment:
Technically this should be `PILImage: ImageModule | Literal[False]`
But I believe the intent is to use it as a boolean.